xmodem/__init__.py

```diff
@@ -377,6 +393,22 @@ def recv(self, stream, crc_mode=1, retry=16, timeout=60, delay=1, quiet=0):
 
         Returns the number of bytes received on success or ``None`` in case of
         failure.
+
+        :param stream: The stream object to write data to.
+        :type stream: stream (file, etc.)
+        :param crc_mode: XMODEM CRC mode
+        :type crc_mode: int
+        :param retry: The maximum number of times to try to resend a failed
+                      packet before failing.
+        :type retry: int
+        :param timeout: The number of seconds to wait for a response before
+                        timing out.
+        :type timeout: int
+        :param delay: The number of seconds to wait between resend attempts
+        :type delay: int
+        :param quiet: If ``True``, write transfer information to stderr.
+        :type quiet: bool
```

Comment:
FYI you can save a lot of lines, by using:
```
:param int delay: The number of seconds....
```


Rather than,
```
:param delay: The number of seconds ...
:type delay: int
```

But its no big deal, accepted as-is